controllers/machinedeletionremediation_controller_test.go

```diff
@@ -143,11 +143,11 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 					verifyMachineNotDeleted(workerNodeMachineName)
 					verifyMachineNotDeleted(masterNodeMachineName)
 					verifyConditionsMatch([]expectedCondition{
-						{commonconditions.ProcessingType, metav1.ConditionFalse, remediationSkippedNodeNotFound},
-						{commonconditions.SucceededType, metav1.ConditionFalse, remediationSkippedNodeNotFound}})
+						{commonconditions.ProcessingType, metav1.ConditionFalse, remediationCannotStartNodeNotFound},
+						{commonconditions.SucceededType, metav1.ConditionFalse, remediationCannotStartNodeNotFound}})
 					verifyConditionUnset(commonconditions.PermanentNodeDeletionExpectedType)
 					verifyEvents([]expectedEvent{
-						{v1.EventTypeWarning, "RemediationSkippedNodeNotFound", "failed to fetch node", true},
+						{v1.EventTypeWarning, "RemediationCannotStart", "Could not get remediation target Node", true},
```

Comment:
Didn't you mean to switch `RemediationSkippedNodeNotFound`  to `remediationCannotStartNodeNotFound` instead of `RemediationCannotStart`?

```suggestion
						{v1.EventTypeWarning, remediationCannotStartNodeNotFound, "Could not get remediation target Node", true},
```

Reply:
This comes from `common`, where the reason is `RemediationCannotStart`

Reply:
Then maybe we should update Common first. 
It seems weird that there are`remediationCannotStartNoControllerOwner`, and `RemediationCannotStartMachineNotFound` event reasons, which are straightforward, and instead of the old `RemediationSkippedNodeNotFound` reason we use generic `RemediationCannotStart` instead of `remediationCannotStartNodeNotFound`. All Medik8s remediator operators work with nodes, and this generic event is meant for when the node is missing and then remediation cannot start.

Reply:
> we use generic `RemediationCannotStart`

I think it's good that is generic because it's in `common` :shrug:

Reply:
But why be generic if being more specific still fits everyone? The only downside for this is if we change the CRs API and logic then we might not have this kind of event. If so, then a new event should be created. That seems ok to me.

I think it would make sense to use `remediationCannotStartNodeNotFound` in common, so MDR, FAR, SNR (and NHC) could use the same event which would be straightforward, and with a better meaning IMO. I don't see another RemediationCannotStart kind of scenario that fits all the operators, if there is one that could fit some of them and we want to use this event for this use case then it would be fine. But having a more concrete event reason still seems better to me.

Reply:
There is still the message "Could not get remediation target Node" to clarify.
I'd say, I could use the same reason "remediationSkipped" for the other MDR's event and then clarify either "missing controller owner" or "missing target node"

Reply:
The `common` repo could instead get it's reasons as variables so that the operators can reuse it (e.g. `common.RemediationCannotStart`)